app/views/search.scala.html

```diff
@@ -50,11 +51,17 @@
 	if(raw.contains("multiVolumeWork")) {select(("Band","Bände"))} else if (raw.contains("series")) {select(("Serienband","Serienbände"))} else if (raw.contains("containedIn")) {select(("Beitrag","Beiträge"))} else {""}
 }
 
-@nwbibLabel() = @{
-	if(!nwbibsubject.isEmpty && !nwbibsubject.contains(",") && Seq(nwbibspatial, person, name, subject, id, publisher, issued, medium, owner, t, set, location, word, corporation, raw).forall(_.isEmpty)){
-		s" in Sachsystematik zu '${Lobid.facetLabel(Seq(nwbibsubject),"","")}'"
```

Comment:
Don't know if true here, but often it's due to either some formatting requirement in the template (line breaks becoming line breaks in the resulting HTML), or a requirement of the embedded scripts, which can get invalid syntax with some line breaks. I don't like these long lines either, so feel free to try adding line breaks locally when you encounter things like this in reviews, would be great to get rid of some of these.